### ISSUE TITLE
Identify reason for 400 calorie range plan option being off

### ISSUE THREAD

**Issue by ismu1** [sanctuarycomputer/smalls#36]
This is a bigger issue where the Smalls team do not understand or can identify the logic behind the intelligence. There's a cut off in the system that makes the customer buy either 11.79 or 0.42 (days of too much food) where a cat is in need of 400 calories and one of them is perfect and the other option gives the customer a package with 11 days of too much food. Smalls team want to understand why this is happening and the logic behind it, and how to adjust this so ideally the customer will only receive +1, -1 day too much food. 

Here is the document. Focus on the orange fields (row 18-24), and field 18U is where you see the 11.79.
https://docs.google.com/spreadsheets/d/1zBnUYZX67uqpUILF0MYae7r5NxaFHhwe5dT9WvtdR-k/edit#gid=0

This seems like a big issue, but let's sit and I can walk you through the Excel calorie document where the issue is occurring.

**Comment by joshiefishbein** [comment]
Need example of this happening

**Comment by ismu1** [comment]
From Matt: So, the basic problem here is that we would like to have a threshold of +/- 2 days of packets being sent per shipment so that we are never sending way too much or way too little.

Our specification to Vaan was to only allow +/- 2 days of food but this rule doesn't seem to be correctly applying in all cases. In this sheet (https://docs.google.com/spreadsheets/d/1zBnUYZX67uqpUILF0MYae7r5NxaFHhwe5dT9WvtdR-k/edit?usp=sharing), we highlight in orange and purple places where this is not the case. The purple highlight is the place where we are most concerned because it represents a cumulative cat calorie requirement that is quite common.

To reproduce this, go through the on-boarding flow and enter that you have 2 cats. Make one cat about 10lbs and another cat 12lbs and play around until you see the together the two cats need 475 calories.

Once you get this done, proceed to the stage where you select your shipping frequencies. Here you will see three options. The most frequent of which represents the trouble area because shipping that number of packets with that frequency would result in absolute value greater than +/- two days of food for the user.

This isn't a major problem because the option is pretty uneconomical for the user since they would be getting way more food than they need so its likely that users never select this option.

My primary goal for this ticket is to use it as a vehicle to better understand the logic underlying out existing system so that we can more easily manipulate variables, like tolerance for too much or too little food, in the future.

My secondary goal is to slightly adjust the tolerance so that the areas highlighted in orange can be removed from the options that users see. That said, there will likely be compromises to not showing those options so I'd like to first understand what options the users would be shown should we find a way to reduce the tolerance.

TLDR: We don't really understand all of the logic that goes into showing users the options they are allowed to select. We'd like to better understand that. Tolerance for too much or too little food is a good one to start with, I hope, and therefore this is worth investigating.
